pkg/asset/installconfig/installconfig.go

```diff
@@ -75,6 +79,11 @@ func (a *InstallConfig) Generate(parents asset.Parents) error {
 		PullSecret: pullSecret.PullSecret,
 	}
 
+	if a.Config.Networking == nil {
+		a.Config.Networking = &types.Networking{}
+	}
+	a.Config.Networking.MachineNetwork = networking.machineNetwork
```

Comment:
Fixed it with adding the following to the InstallConfig structure creation (instead of `Networking.MachineNetwork: networking.machineNetwork,`):
```
Networking.MachineNetwork: networking.machineNetwork,
       Networking.MachineNetwork: networking.machineNetwork,
},
```